external/opensocdebug/software/src/osd-private.h

```diff
@@ -125,11 +126,11 @@ static const uint8_t REG_WRITE32 = 0x5;
 static const uint8_t REG_WRITE64 = 0x6;
 static const uint8_t REG_WRITE128 = 0x7;
 
-static const uint16_t modules_max_id = 4;
+static const uint16_t modules_max_id = 7;
 struct module_types {
     const char *name;
 };
-extern const struct module_types module_lookup[6];
+extern const struct module_types module_lookup[7];
 
```

Comment:
Solved. modules_max_id was changed to a parameter.
The parameter is used for module_lookup and for the function osd_get_module_name